draft-ietf-scitt-architecture.md

```diff
@@ -1095,3 +1106,194 @@ To indicate that the content is an scitt configuration represented as JSON:
+
+All SCITT identifiers share common parameters to promote interoperability:
+
+Let hash-name be an algorithm name registered in {{IANA.named-information}}.
+
+To promote interoperability, the hash-name MUST be "sha-256".
+
+Let base-encoding, be a base encoding defined in {{-Base64Url}}.
+
+To promote interoperability, the base encoding MUST be "base64url".
+
+In the blocks and examples that follow, NOTE: '\' line wrapping per RFC 8792.
+
+## For Binary Content
+
+Identifiers for binary content, such as Statements, or even Artifacts themselves are computed as follows:
+
+Let the `base64url-encoded-bytes-digest` for the messsage be the base64url encoded digest with the chosen hash algorithm of bytes / octets.
+
+Let the SCITT name for the message be the URN constructed from the following URI templatem, according to {{-URITemplate}}:
+
+Let the `message-type`, be "statement" for Statements and Artifacts.
+
+~~~
+urn:ietf:params:scitt:\
+{message-type}:\
+{hash-name}:{base-encoding}:\
+{base64url-encoded-bytes-digest}
+~~~
+
+## For SCITT Messages
+
+Identifiers for COSE Sign 1 based messages, such as identifiers for Signed Statements and Receipts are computed as follows:
+
+Let the `base64url-encoded-to-be-signed-bytes-digest` for the messsage be the base64url encoded digest with the chosen hash algorithm of the "to-be-signed bytes", according to {{Section 8.1 of RFC9052}}.
+
+Let the SCITT name for the message be the URN constructed from the following URI templatem, according to {{-URITemplate}}:
+
+Let the `message-type`, be "signed-statement" for Signed Statements, and "receipt" for Receipts.
+
+~~~
+urn:ietf:params:scitt:\
+{message-type}:\
+{hash-name}:{base-encoding}:\
+{base64url-encoded-to-be-signed-bytes-digest}
+~~~
+
+Note that this means the content of the signature is not included in the identifer, even though signature related claims, such as activation or expiration information in protected headers are included.
+
+As a result, an attacker may construct a new signed statement that has the same identifier as a previous signed statement, but has a different signature.
+
+## For Transparent Statements
+
+Identifiers for Transparent Statements are defined as identifiers for binary content, but with "transparent-statement" as the `message-type`.
+
+~~~
+urn:ietf:params:scitt:\
+{message-type}:\
+{hash-name}:{base-encoding}:\
+{base64url-encoded-bytes-digest}
+~~~
+
+Note that because this identifier is computed over the unprotected header of the Signed Statement, any changes to the unprotected header, such as changing the order of the unprotected header map key value pairs, adding additional receipts, or adding additional proofs to a receipt, will change the identifier of a transparent statement.
+
+Note that because this identifier is computed over the signatures of the signed statement and signatures in each receipt, any cannonicalization of the signatures after the fact will produce a distinct identifier.
```

Comment:
@fournet @ad-l if you have use cases for this, please suggest them here.

If you have security concerns with this, please also leave suggestions.

Reply:
As there are open suggestions anymore and the feedback from @fournet was addressed by @OR13, I am moving on with approval and merge.